lbrynet/core/Wallet.py

```diff
@@ -654,9 +618,15 @@ def claim_name(self, name, bid, metadata):
 
         claim = self._process_claim_out(claim)
         claim_outpoint = ClaimOutpoint(claim['txid'], claim['nout'])
-        log.info("Saving metadata for claim %s %d", claim['txid'], claim['nout'])
-
-        yield self._save_name_metadata(name, claim_outpoint, _metadata['sources']['lbry_sd_hash'])
```

Comment:
This is wrong. Firstly because generate_claimid is broken because it hashes a hex encoded txid and not txid bytes. 
You also don't need to generate claim id because when it is a new claim, the lbryum claim command returns the claim_id in the output, and if it is not a new claim, you already know the old claim_id in my_claim. 

        if is_new_claim:
            claim_id = claim['claim_id']
        else:
            claim_id = my_claim['claim_id']